lesson_16/objects/objects_app/src/test/java/com/codedifferently/lesson16/xaviercruz/PersonTest.java

```diff
@@ -0,0 +1,149 @@
+package com.codedifferently.lesson16.xaviercruz;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import org.junit.jupiter.api.Test;
+
+public class PersonTest {
+  Person person =
+      new Person(
```

Comment:
Should use `beforeEach` to reset the class under test so that the state from one test to the next isn't mutated in ways you don't expect.

Reply:
Done